src/AddTodoForm.jsx

```diff
@@ -1,9 +1,18 @@
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+
+    const handleAddTodo = (event) => {
+        const titleValue = event.target.title.value
+        event.preventDefault()
+        const todoTitle = titleValue
+        props.onAddTodo(todoTitle)
```

Comment:
rather than do this:

```
const todoTitle = titleValue
props.onAddTodo(todoTitle)
```
You can simply write:
```
...
const todoTitle = event.target.title.value
...

props.onAddTodo(todoTitle)
```
Adding the extra variable assignment only makes your code verbose.

Also, two extra things you need to do as per this week's assignment requirement are to:
- [ ] Log the value of `todoTitle`
- [ ] Reset the form after submission.

Reply:
Done